uart_sssss.c

```diff
@@ -54,3 +54,29 @@ void uart2_transmit(int ch)
 ///
 ///
 
+
```

Comment:
set macros to the bits in order for the colleagues to read and understand better the code without the need for them to go to the datasheet

Reply:
done

Reply:
dsafasdfsadfsadfsadf